docs/reference-guides/block-api/block-transforms.md

```diff
@@ -341,7 +341,7 @@ transforms: {
 
 Via the optional `transforms` key of the block configuration, blocks can use the `ungroup` subkey to define which inner blocks of a grouping block can be extracted, replacing the block.
 
-If a block has an `ungroup` transform, it is considered ungroupable, without the requirement of being the default grouping block. The UI used to ungroup a block with this API, is the same with the one used for the default grouping block. In order for the ugroup UI to render, we must have a single grouping block selected, which also contains some inner blocks.
+If a block has an `ungroup` transform, it is considered ungroupable, without the requirement of being the default grouping block. The UI used to ungroup a block with this API is the same as the one used for the default grouping block. In order for the Ungroup button to be displayed, we must have a single grouping block selected, which also contains some inner blocks.
```

Comment:
> it is considered ungroupable

I think this phrase might be confusing for folks because "ungroupable" means "Incapable of being grouped together".

Reply:
Agreed. Depending on the use in a sentence, some ideas: "able to be ungrouped", "ungroup-ready", "eligible for ungrouping"? 😅

Reply:
You're right. I went for `eligible for ungrouping`.